### PR TITLE
Todo list

### PR DESCRIPTION
**Pull request by marcelomaidden**
## Todo list project
At this pull request I made a Todo project with all functionalities below:
- Create todo with title, description, priority, complete check and, due date
- Create projects to group Todos
- Change background-color depending on Todo priority
- Change background to gray when Todo is marked as complete
- Delete todo
- Separate files for different classes (e.g. Todo, Project, Projects, TodoList)
- Use WebPack to create a bundle js file
- Create WebPack configuration to bundle js file
- Add scripts to package.json in order to run webpacks' build and watch

### CODE REVIEW

File: src/index.js
Comment (christianotieno) [inline src/index.js]: ## Required change 
#### You need to make sure you are using ES6 syntax as described in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)

- [ ] In this instance, you ought to have employed the usage of an arrow function instead of this anonymous `function` expression.

File: src/index.js
Comment (christianotieno) [inline src/index.js]: ## Required change 
#### You need to make sure you are using ES6 syntax as described in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)

- [ ] In this instance, you ought to have employed the usage of an arrow function instead of this anonymous `function` expression.

File: src/index.js
Comment (christianotieno) [inline src/index.js]: ## Fix todo creation hitch on new projects.
- [ ] A user is unable to create their todos on newly created projects saved for the `default` project as shown below 👇 
to-do-list
. Kindly fix this bug.

File: src/index.js
Comment (christianotieno) [inline src/index.js]: ## Required change 
#### You need to make sure you are using ES6 syntax as described in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)

- [ ] In this instance, you ought to have employed the usage of an arrow function instead of this anonymous `function` expression.

File: src/index.js
Comment (christianotieno) [inline src/index.js]: ## Required change 
#### You need to make sure you are using ES6 syntax as described in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)

- [ ] In this instance, you ought to have employed the usage of an arrow function instead of this anonymous `function` expression.

File: src/index.js
Comment (christianotieno) [inline src/index.js]: ## Required change 
#### You need to make sure you are using ES6 syntax as described in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)

- [ ] In this instance, you ought to have employed the usage of an arrow function instead of this anonymous `function` expression.

File: src/index.js
Comment (christianotieno) [inline src/index.js]: ## Required change 
#### You need to make sure you are using ES6 syntax as described in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)

- [ ] In this instance, you ought to have employed the usage of an arrow function instead of this anonymous `function` expression.

File: src/index.js
Comment (christianotieno) [inline src/index.js]: ## Required change 
#### You need to make sure you are using ES6 syntax as described in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)

- [ ] In this instance, you ought to have employed the usage of an arrow function instead of this anonymous `function` expression.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @marcelomaidden 👋 
Congratulations on the hard work done in this project. I love how clean and modular your code is. Keep it up. 👏 

There are some changes that I have suggested below, 👇  kindly take care of them before we proceed, thank you. 🙏 

### Happy coding 💻 

#### Christian Otieno(TSE)

**Review by shloch — CHANGES_REQUESTED** [review]
## helo @marcelomaidden 

Great job with the huge improvements from the previous review. The color picker is really cool. I will need ONE more task from you, to get a thumbs up. Prevent the validation of "empty" forms. I could try to add a `project` and submit an empty form, the app will validate. Same for `todo`
todo

Waiting for a final review
Happy coding

**Review by bellom — APPROVED** [review]
## Hi @marcelomaidden :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**
